e2e/discussions.spec.ts

```diff
@@ -226,28 +255,32 @@ test.describe('Discussions', () => {
               }),
             ).toBeVisible()
             await expect(discussionThread.getByText('Moderator')).toBeVisible()
-            await expect(discussionThread.getByText(threadTitle)).toBeVisible()
-            await expect(discussionThread.getByText(threadBody)).toBeVisible()
           })
 
           return threadId
         },
       )
 
       await testAuth.step('View count is updated', async () => {
-        await userPage.getByRole('button', { name: 'Show All Threads' }).click()
-
-        // reload is necessary for view count to update
-        await userPage.reload()
+        await userPage
+          .getByRole('button', { name: /show all threads/i })
+          .click()
 
         await expectDiscussionPageLoaded(userPage, userProject.id)
-        await expectThreadTableLoaded(
-          userPage,
-          threadTitle,
-          userConfigs['swc-e2e-user'].username,
-          '0',
-          '1',
-        )
+
+        // retry if the view count hasn't updated
+        await expect(async () => {
+          // reload is necessary for view count to update
+          await userPage.reload()
+          await expectDiscussionPageLoaded(userPage, userProject.id)
+          await expectThreadTableLoaded(
+            userPage,
+            threadTitle,
+            userConfigs['swc-e2e-user'].username,
+            '0',
+            '1',
+          )
+        }).toPass()
```

Comment:
The test intermittently failed because the view count wasn't updated, so retry the reload to give the view count more time to update

Reply:
I don't think the view count is that important, so if it keeps flaking, then we should remove those checks IMO